torch/__init__.py

```diff
@@ -1320,11 +1320,8 @@ class _TorchCompileInductorWrapper:
     compiler_name = "inductor"
 
     def __init__(self, mode, options, dynamic):
-        from torch._inductor.compile_fx import compile_fx
-
-        self.compile_fn = compile_fx
-        self._torchdynamo_orig_callable = compile_fx
         self.config = dict()
+        self.dynamic = dynamic
```

Comment:
why is dynamic not in config?

Reply:
Config here is inductor config, and dynamic is not part of inductor's config.  (It controls dynamo behavior.)